fixtures/sources/Greeter.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.13;
+
+contract Greeter {
```

Comment:
Mentioned in [issue](https://github.com/ChainSafe/web3.js/issues/4850) and discussed in backlog grooming discussion to write tests for all functions of erc20 and erc721

Reply:
Will cover this in another issue. As current PR already contains a lot of changes.